diff --git a/node_modules/react-vnc/dist/index.es.js b/node_modules/react-vnc/dist/index.es.js
index 5d56389..60bbb7b 100644
--- a/node_modules/react-vnc/dist/index.es.js
+++ b/node_modules/react-vnc/dist/index.es.js
@@ -13687,6 +13687,7 @@ class RFB extends EventTargetMixin {
     _negotiateXvpAuth() {
         if (this._rfbCredentials.username === undefined ||
             this._rfbCredentials.password === undefined ||
+            this._rfbCredentials.password === null ||
             this._rfbCredentials.target === undefined) {
             this.dispatchEvent(new CustomEvent(
                 "credentialsrequired",
diff --git a/node_modules/react-vnc/dist/index.js b/node_modules/react-vnc/dist/index.js
index 5f9d4b3..484ea0c 100644
--- a/node_modules/react-vnc/dist/index.js
+++ b/node_modules/react-vnc/dist/index.js
@@ -13691,6 +13691,7 @@ class RFB extends EventTargetMixin {
     _negotiateXvpAuth() {
         if (this._rfbCredentials.username === undefined ||
             this._rfbCredentials.password === undefined ||
+            this._rfbCredentials.password === null ||
             this._rfbCredentials.target === undefined) {
             this.dispatchEvent(new CustomEvent(
                 "credentialsrequired",
